roadmap_schedule.md

```diff
@@ -0,0 +1,17 @@
+# Roadmap Schedule
+
+## Вівторок
+
+C++/Qt branch: 	 **18:00**, [Природничий коледж ЛНУ, 129 ауд](http://bit.ly/2djyiNv)
+
+## Середа
+
+Web branch: 		**18:30**, [Головний корпус ЛНУ, 111 ауд](http://bit.ly/2dsuxGB)
```

Comment:
Please, remove line about web - for now web could not be held on Prykladna (Main campus)